src/Raven.Studio/typescript/viewmodels/shell/about.ts

```diff
@@ -183,11 +183,25 @@ class about extends viewModelBase {
            return "icon-cancel";
         });
     }
+    
+    renewLicenseConfigurationSettings = ko.observable<boolean>(false);
+    activateLicenseConfigurationSettings = ko.observable<boolean>(false);
+    forceLicenseUpdateConfigurationSettings = ko.observable<boolean>(false);
 
     register() {
         registration.showRegistrationDialog(license.licenseStatus(), false, true);
     }
 
+    private getLicenseConfigurationSettings() {
+        new getLicenseConfigurationSettingsCommand()
```

Comment:
you don't return this promise, but you await for that in $.when

Reply:
Added 'return'